### PR TITLE
Rewrite LinuxIDMappings

### PR DESCRIPTION
**Pull request by hqhq**
Basicly make the format consistent with others, no
semantics change.

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: config-linux.md [hunk 1]
```diff
@@ -73,6 +73,18 @@ If a new namespace is not created (because the namespace type is not listed, or
 
 ## User namespace mappings
 
+**`uidMappings`** (array of objects, OPTIONAL) describes the user namespace uid mappings from the host to the container.
+**`gidMappings`** (array of objects, OPTIONAL) describes the user namespace gid mappings from the host to the container.
+
+The following parameters can be specified:
```
Comment (hqhq) [inline config-linux.md]: @wking I've push a new commit to make them consistent.

**Comment by wking** [comment]
Through 4404abf6 looks good to me.

**Comment by crosbymichael** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**